### PR TITLE
Prepare Readme for Inital Release

### PR DESCRIPTION
**Pull request by JamesCropcho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,71 +119,82 @@ app.on('export-error', () => {})
...
-## Other useful links
+Dispatch to one component based on the URL of the request:
+
+`curl localhost:9090/plotly-graph/ <payload>`
+
+…or e.g.:
+
+`curl localhost:9090/plotly-dashboard/ <payload>`
+
+### Pixel Comparisons
```
Comment (etpinard) [inline README.md]: Oh that section is obsolete. :hocho:

File: README.md [hunk 2]
```diff
@@ -1,79 +1,56 @@
...
+# Plotly Image Exporter
 
-aka image server 2.0.
+## Install
 
-See https://github.com/plotly/streambed/issues/9655 for info on the requirements.
+### Dependencies
 
-## Install
+The environment you're installing this into may require Poppler.
```
[the remark this answers is not in the document]
Comment (JamesCropcho) [reply README.md]: I have made the requested change: https://github.com/plotly/image-exporter/blob/prepare-readme-for-inital-release/README.md

**Comment by etpinard** [comment]
First off, @JamesCropcho thanks for taking this on!

A few suggestions:

- the `Logging` section is obsolete
- the `Other Useful Links` is also obsolete
- I would move the `Electron`, `Components` and `Nomenclature` sections into a new `CONTRIBUTING.md` file. Users don't need to know about these things.
- the install section should feature links to the packaged executables just like on https://github.com/plotly/falcon-sql-client
- the CLI section should be above the API section.

File: README.md [hunk 3]
```diff
@@ -1,79 +1,56 @@
-# image-exporter
+# Plotly Image Exporter
 
-aka image server 2.0.
+## Install
 
-See https://github.com/plotly/streambed/issues/9655 for info on the requirements.
```
Comment (cpsievert) [inline README.md]: IMO, this should say `npm install -g` (the downstream examples are assuming this)

File: README.md [hunk 4]
```diff
@@ -142,71 +119,78 @@ app.on('export-error', () => {})
...
-# Dispatch to one component based on url of request
-curl localhost:9090/plotly-graph/ <payload>
+#### R Usage Example
 
-# or e.g.
-curl localhost:9090/plotly-dashboard/ <payload>
+```R
+library(plotly)
+system2("plotly-graph-exporter", plotly_json(fig, FALSE))
 ```
```
Comment (cpsievert) [inline README.md]: Unfortunately, it won't be this simple. Eventually we should provide a wrapper in the R package for this, but for now, you could use this R code:

```r
library(plotly)
p <- plot_ly(x = 1:10, y = 1:10, color = 1:10)
b <- plotly_build(p)$x[c("data", "layout")]
json <- plotly:::to_JSON(b)
cmd <- sprintf("plotly-graph-exporter '%s' -o r-export-test.png", json)
system(cmd)
```

File: README.md (hunk 3 above)
Comment (etpinard) [reply README.md]: Related to what was discussed in https://github.com/plotly/image-exporter/pull/73#issuecomment-381774163, the downstream examples are assuming that use of the standalone app.

File: README.md (hunk 4 above)
Comment (etpinard) [reply README.md]: Thanks for the tip!

**Comment by cpsievert** [comment]
I'd love to see a section explaining what output files are supported and how this tool compares to [Plotly.downloadImage](https://plot.ly/javascript/plotlyjs-function-reference/#plotlydownloadimage)

File: README.md (hunk 4 above)
Comment (etpinard) [reply README.md]: done in https://github.com/plotly/image-exporter/blob/b8d52dc502c56dceba6b31fd57be47b9c2e3fd2b/README.md

**Pull request merged**